index.js

```diff
@@ -26,7 +28,7 @@ export async function run() {
 export async function setupRuby(options = {}) {
   const inputs = { ...options }
   for (const key in inputDefaults) {
-    if (!inputs.hasOwnProperty(key)) {
+    if (!Object.prototype.hasOwnProperty.call(inputs, key)) {
```

Comment:
Is this needed? What does it do differently?

Reply:
This was an ESLint thing.  I still use it locally to (hopefully) catch typo's etc.  I need a new keyboard, as my old one required a lot less travel.